dts/bindings/battery/battery.yaml

```diff
@@ -0,0 +1,84 @@
+#
+# Copyright 2022 Google LLC
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+description: |
+   Battery parameters and info
+
+   Voltage is in millivolts
+   Current is in milliamperes
+   Temperature is in Celsius degrees
+
+compatible: "zephyr,battery"
+
+properties:
+   manuf_name:
+      description: Manufacturer name
+      type: string
+   device_name:
+      description: Model/Device name
+      type: string
+   voltage_max:
+      description: |
+         Maximum voltage that can be applied to the battery.
+      type: int
+   voltage_normal:
```

Comment:
voltage_nominal rather than voltage_normal? Or the comment to match voltage_normal would make sense.